src/CodeGen/ODataT4CodeGenerator.ttinclude

```diff
@@ -1397,6 +1397,8 @@ public abstract class ODataClientTemplate : TemplateBase
     internal void WriteEntityContainer(IEdmEntityContainer container, string fullNamespace)
     {
         string camelCaseContainerName = container.Name;
+		string path = this.context.TempFilePath; 
+		bool useTempFile = !String.IsNullOrEmpty(path) && System.IO.File.Exists(path);

```

Comment:
>System.IO.File.Exists(path) [](start = 52, length = 27)

Should we add this here? We should be able to create a new file if there is no file in the path.

Reply:
This should not be added. Modified accordingly.